pkg/skaffold/schema/latest/config.go

```diff
@@ -67,6 +67,9 @@ type Pipeline struct {
 
 	// PortForward describes user defined resources to port-forward.
 	PortForward []*PortForwardResource `yaml:"portForward,omitempty"`
+
+	// Debug describes how images are deployed.
+	Debug DebugConfig `yaml:"debug,omitempty"`
```

Comment:
Introducing a new top level config might be a bit more complicated than that. We might have to check that the yaml documentation is ok, especially for the `profiles` section that can be used to override this.